stdlib/public/Platform/winsdk.modulemap

```diff
@@ -377,5 +377,11 @@ module WinSDK [system] {
 
     link "AdvAPI32.Lib"
   }
+
+  module WLANAPI {
+    header "wlanapi.h"
```

Comment:
According to MSDN this header requires linking against `wlanapi.lib`, perhaps it should be specified in this submodule?

Reply:
ooh, I was looking for the library to link against, where did you find that?  I think that it should.

Reply:
I just opened a doc page for a function from this header, and looked under the "Requirements" section:
https://docs.microsoft.com/en-us/windows/win32/api/wlanapi/nf-wlanapi-wfdcancelopensession#requirements

Reply:
Ah, I wonder what I had looked at instead - the process is the same one that I took, might've been an inline function or macro (bad luck) *shrug*

Reply:
Gotta hope there isn't a rogue function somewhere in one of the headers used by WinSDK, which requires linking against a different library then the rest of the header :)

Reply:
No, it was that it listed no library to link against instead - which is why I didn't have one in the first place.